### PR TITLE
Modify README file

### LINKED ISSUE #1482: Modify the README file
### What would you like to be enhanced?

Modify the quick start steps in the README file

### Why is this needed?

Adapts to the latest Sermant version.

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**
Document

**What this PR does / why we need it?**

Modify the quick start steps in the README file.

**Which issue(s) this PR fixes？**

Fixes #1482

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: README-en.md [hunk 1]
```diff
@@ -44,22 +44,13 @@ Below is a simple demo that guides new users to use Sermant in just 4 steps.
 
 ### Preparation
 
-- [Download](https://github.com/huaweicloud/Sermant/releases/download/v1.2.1/sermant-1.2.1.tar.gz) Sermant package (The current version recommended is 1.2.1)
-- [Download](https://github.com/huaweicloud/Sermant-examples/tree/main/flowcontrol-demo/spring-cloud-demo/spring-provider) demo application
+- [Download](https://github.com/huaweicloud/Sermant/releases/download/v1.4.0/sermant-1.4.0.tar.gz) Sermant Release package (The current version recommended is 1.4.0)
+- [Download](https://github.com/huaweicloud/Sermant-examples/releases/download/v1.4.0/sermant-examples-flowcontrol-demo-1.4.0.tar.gz) demo binary product compression package
```
Comment (daizhenyu) [inline README-en.md]: Demo uppercase，same below

File: README.md [hunk 2]
```diff
@@ -84,7 +74,7 @@ java -jar sermant-backend-x.x.x.jar
 
 ### 启动demo应用
```
Comment (daizhenyu) [inline README.md]: Demo

File: README-en.md (hunk 1 above)
Comment (hanbingleixue) [reply README-en.md]: resolved

File: README.md (hunk 2 above)
Comment (hanbingleixue) [reply README.md]: resolved

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**